webbpsf/tests/test_opds.py

```diff
@@ -22,3 +29,49 @@ def test_enable_adjustable_ote():
 
     assert ote.rms() > rms*10, "Huge piston offset didn't make the WFE much worse"
 
+
+
+GLOBAL_FOCUS = [-0.018251043541410904e-08]
+GLOBAL_FOCUS_SCALED = [-0.018251043541410904e-08 * 0.5]
+COEFFS_A1 = np.array([-3.52633363e-09, -2.90050902e-09, 1.25432196e-09, -7.43319098e-12,
+                      -5.82462948e-11, -1.27115922e-10, -1.91541104e-12, 3.64760396e-11,
+                      4.97176630e-13])
+thermal_model_parameters = ([1 * u.day, 'SM', GLOBAL_FOCUS, None],
+                            [1 * u.day, 'SM', GLOBAL_FOCUS_SCALED, .5],
+                            [1440, 'SM', GLOBAL_FOCUS, None],
+                            [1 * u.day, 'A1', COEFFS_A1, None],
+                            [0.0 * u.day, 'SM', [0.0], None],
+                            [0.0 * u.day, 'A1', np.zeros(9), None],
+                            [1.0 * u.day, 'D1', [0.0], None])
+@pytest.mark.parametrize('time, seg, coeff_truth, scaling', thermal_model_parameters)
+def test_thermal_slew_opd(time, seg, coeff_truth, scaling):
+    """ Test that the OTE Thermal model is outputting the correct values """
+    delta_time = time
+    # Create the thermal model
+    otelm = webbpsf.opds.OTE_Linear_Model_WSS()
+    otelm.delta_time = delta_time
```

Comment:
This part might become a little more elegant if you instead did this via 
```
otelm.thermal_slew(delta_time, start_angle, end_angle, scaling)
``` 
That'd be more like how we expect end-users to interact with this code, right? By calling that function, rather than by individually assigning values to attributes of the OTE linear model one at a time. In some sense it's a nicer test if we interact with the code the same way a user would. 

(To be clear, what you're doing here works perfectly well too.)

Reply:
This is a good point. I had somehow convinced myself that this wouldn't work, but it does. I agree that they way you have described is more how we expect users to use it. I will change it also because it's simpler code.